src/pex-builder/build-builder.sh

```diff
@@ -40,7 +40,6 @@ $VENV_ROOT/bin/pipenv requirements --exclude-markers > build/lib/requirements.tx
 # Use a PEX_ROOT to completely isolate this build from others that may be on this machine
 export PEX_ROOT=build/.pex
 $VENV_ROOT/bin/pex -r build/lib/requirements.txt -D build/lib -o $BUILDER_PEX_PATH -v --include-tools \
-    --python=python3.8 \
```

Comment:
This links in all the dependencies for the local platform, which on my m1 mac is painful (see grpcio).
We only need to run this in CI/CD and can still run this with rosetta Python on macs, so this is not essential.